v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -90,8 +126,9 @@ const processTrackEvent = async (metadata, message, destination) => {
   payload.conversionAdjustments[0].adjustmentType = "ENHANCEMENT";
   // Removing the null values from userIdentifier
   const arr = payload.conversionAdjustments[0].userIdentifiers;
-  payload.conversionAdjustments[0].userIdentifiers = arr.filter(() => {
-    return true;
+  payload.conversionAdjustments[0].userIdentifiers = arr.filter(item => {
```

Comment:
Why not use removeUndefinedAndNull values to do this?

Reply:
This function is converting the userIdentifiers from object to araay.